TrafficSimulator/Assets/RoadGenerator/Core/Editor/RoadSystemEditor.cs

```diff
@@ -7,29 +7,61 @@ namespace RoadSystemGenerator
     [CustomEditor(typeof(RoadSystem))]
     public class RoadSystemEditor : Editor 
     {
-        private SerializedProperty showRoadSystemGraphButton;
+        #region SerializedProperties
+            private SerializedProperty _drivingSide;
+            private SerializedProperty _spawnRoadsAtOrigin;
+            private SerializedProperty _showGraph;
+        #endregion
         public void OnEnable()
         {
-            showRoadSystemGraphButton = serializedObject.FindProperty("ShowGraph");
+            _showGraph = serializedObject.FindProperty("ShowGraph");
+            _spawnRoadsAtOrigin = serializedObject.FindProperty("SpawnRoadsAtOrigin");
+            _drivingSide = serializedObject.FindProperty("DrivingSide");
+
         }
         public override void OnInspectorGUI()
         {
-            DrawDefaultInspector();
+            serializedObject.Update();
+            // Uncomment this if you want to change the connections to containers and prefabs
+            
+            //DrawDefaultInspector();
+            
             RoadSystem roadSystem = (RoadSystem)target;
-            showRoadSystemGraphButton.boolValue = EditorGUILayout.Toggle("Show Road System Graph", roadSystem.ShowGraph);
-            // Set the ShowGraph property and update the road system graph if the button does not correspond to the ShowGraph property
-            if (showRoadSystemGraphButton.boolValue != roadSystem.ShowGraph)
+            
+            bool changed = false;
+
+            //_showGraph.boolValue = EditorGUILayout.Toggle("Show Road System Graph", roadSystem.ShowGraph);
```

Comment:
why is this a comment?

Reply:
I left it during development and forgot to remove it.
Removed in 4f8c94910af37ed0a901788cb60e14355014e44d